### ISSUE TITLE
More interesting default code

### ISSUE THREAD

**Issue by isaac-dunn** [cambridge-alpha-team/unvisual-frontend#81]
Are we happy with just having a repeated beep as the default?

**Comment by LauraneS** [comment]
What would you suggest instead? I think this is good because it is basic.

**Comment by isaac-dunn** [comment]
Yeah it's good because it's basically a blank canvas to work from.

Might we want the option to start with something that sounds good, though? Like one of the Sonic Pi examples.

**Comment by LauraneS** [comment]
It would be nice I think. 
Maybe we could have an option to _add_ an example?

**Comment by isaac-dunn** [comment]
How hard would it be to hardcode a tree structure that overwrites the current one when a certain key combination is pressed? As in, a key combination is pressed, and then the code is suddenly some example. Another combination could give the almost blank one that we're using at the moment.

**Comment by RusticFlare** [comment]
I think it is good to have the simple piece of music and give them the tools to create their own complicated one.

**Comment by isaac-dunn** [comment]
I think calling the repeated beep we have at the moment "music" is fairly dubious, and that's my point :P 

Fair enough though if you think we should keep it. I'm mostly thinking about the demo here...
